networking/README.md

```diff
@@ -0,0 +1,40 @@
+# Robotouille Networking README
```

Comment:
I'd add some common usecase commands so others can copy-paste to get a desired result like
1. server command with option to display screen
2. client command with a specific host to connect to
3. how to replay game data onto the simulator
4. how to render game data into a video